ui/app/components/app/modals/export-private-key-modal/export-private-key-modal.component.js

```diff
@@ -146,7 +146,6 @@ export default class ExportPrivateKeyModal extends Component {
         <span className="account-name">{name}</span>
         <ReadOnlyInput
           wrapperClass="ellip-address-wrapper"
```

Comment:
Nit: It looks like this class is assigned rules only if it's within `account-modal-container`, yet it's also only _used_ in two places, both of which are within `account-modal-container`.

Perhaps it would be simpler to retire this class, and move these rules to `readonly-input` instead?

Reply:
I'm generally for that change, though as it stands `ReadOnlyInput` is currently used in two places inside the export-private-key modal code, and only one has this class applied to its wrapper. the other has different styles for the wrapper.

Reply:
Whoops! I had not noticed that. Good point.

Reply:
Offhand I can't think of a simple way to make this more in-line with convention, so... maybe this can be a problem for another day.